testcase/variable_location.mk

```diff
@@ -0,0 +1,54 @@
+
+not_traced := no
+.KATI_TRACE_ASSIGNMENT = true
```

Comment:
What is this for? I can't find a reference to this variable name in the source code.

Reply:
An old iteration of this changed. Removed.